docs/image-build.md

```diff
@@ -0,0 +1,24 @@
+
+# Kube-compare image build
+
+## Build container
+
+```bash
+make image-build
+```
+
+## Copy binary locally
+
+- One option it to build the binary locally
+
+```bash
+make cross-build
+```
+
+- Another option is to extract the binary from the container
+
+```bash
+docker create --name kube-compare kube-compare:latest
```

Comment:
Could try it out? I think because we dont have any cmd specified in dockfile this will throw an error? 
```
docker create --name kube-compare kube-compare:latest ---
```

Reply:
I tested it with Podman and didn't get any error

Reply:
As long as its just called ContainerFile or DockerFile then it should get picked up